initial-promises/node-server/distributePolicy/1.0/propagatePromises.cf

```diff
@@ -66,55 +108,159 @@ bundle agent propagatePromises
+    # promises, masterfiles folder and licenses to be updated.
+    pass3.(((root_server.propagate_tools_ok)|(!root_server.propagate_tools_ok.promises_propagated.masterfiles_propagated.ncf_local_promises_propagated.ncf_common_promises_propagated)).!(propagate_tools_error|could_not_propagate_promise|could_not_propagate_masterfiles|could_not_copy_license|could_not_propagate_ncf_local_promise|could_not_propagate_ncf_common_promise))::
+      "any" usebundle => rudder_common_report("DistributePolicy", "result_success", "inventory-all@@inventory-all@@00", "Propagate promises", "None", "All files have been propagated");
+    pass3.could_not_propagate_promise::
+      "any" usebundle => rudder_common_report("DistributePolicy", "result_error", "inventory-all@@inventory-all@@00", "Propagate promises", "None", "Cannot propagate policy files");
+
+    pass3.could_not_propagate_files::
+      "any" usebundle => rudder_common_report("DistributePolicy", "result_error", "inventory-all@@inventory-all@@00", "Propagate promises", "None", "Cannot propagate shared files");
+
+    pass3.propagate_tools_error::
+      "any" usebundle => rudder_common_report("DistributePolicy", "result_error", "inventory-all@@inventory-all@@00", "Propagate promises", "None", "Cannot propagate tools");
 
-  reports:
-    could_not_propagate_promise::
-      "@@DistributePolicy@@result_error@@root-DP@@root-distributePolicy@@00@@propagatePromises@@None@@${g.execRun}##${g.uuid}@#Cannot propagate policy files";
+    pass3.could_not_propagate_ncf_local_promise::
+      "any" usebundle => rudder_common_report("DistributePolicy", "result_error", "inventory-all@@inventory-all@@00", "Propagate promises", "None", "Cannot propagate local ncf promises");
 
-    could_not_propagate_tools::
-      "@@DistributePolicy@@result_error@@root-DP@@root-distributePolicy@@00@@propagatePromises@@None@@${g.execRun}##${g.uuid}@#Cannot propagate tools";
+    pass3.could_not_propagate_ncf_common_promise::
+      "any" usebundle => rudder_common_report("DistributePolicy", "result_error", "inventory-all@@inventory-all@@00", "Propagate promises", "None", "Cannot propagate common ncf promises");
 
-    could_not_propagate_masterfiles::
-      "@@DistributePolicy@@result_error@@root-DP@@root-distributePolicy@@00@@propagatePromises@@None@@${g.execRun}##${g.uuid}@#Cannot propagate masterfiles";
+    pass3.could_not_propagate_masterfiles::
+      "any" usebundle => rudder_common_report("DistributePolicy", "result_error", "inventory-all@@inventory-all@@00", "Propagate promises", "None", "Cannot propagate masterfiles");
+
+    pass3.could_not_copy_license::
+      "any" usebundle => rudder_common_report("DistributePolicy", "result_error", "inventory-all@@inventory-all@@00", "Propagate promises", "None", "Cannot copy local license");
 
-    could_not_copy_license::
-      "@@DistributePolicy@@result_error@@root-DP@@root-distributePolicy@@00@@propagatePromises@@None@@${g.execRun}##${g.uuid}@#Cannot copy local license";
```

Comment:
The ID in all of these reports is "inventory-all@@inventory-all@@00" which doesn't match the old ID, but I see you did change it in Makefile. So probably, this just needs regenerating so we can read the diff with the right IDs.